lib/actionable/action.rb

```diff
@@ -64,13 +64,27 @@ def model
 
       alias transactional_model model
 
+      def set_safely_nesting_transactional_model(name = :nothing)
```

Comment:
Do we need the `set_` prefix? It seems like leaving it out would be more consistent with `transactional_model`.

Reply:
I perceive the behavior of this method as being more akin to the `set_model` method than the `transactional_model` method. I kept the `set_` for consistency.

Reply:
Fine with me.